study.md

```diff
@@ -43,15 +47,18 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+From what I gather, 'view' is the current status of a SPA, and view-state is related to ASP.NET.  It preserves page and control values between the client and server.  Routers can help by
+saving some/all of the application state, rather than the server or client.
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+Does not necessitate page refreshes - behaves like a native (offline) app
+State-management is performed client-side, taking pressure off of your server(s)
+
 ```
 
 ## List some technologies used to make SPA's
```

Comment:
Ember, Angular, React, are just a few of the front end frameworks that make this happen.